tests/test_task.py

```diff
@@ -27,7 +27,7 @@ def task():
         ({"v": 1, "va": 3, "a": 6}, False, {"v": 0.1, "va": 0.3, "a": 0.6}, OrderedDict),
     ],
 )
-def test_init_session(
+def test_post_init_session(
```

Comment:
To refer to the `post_init` method. It's arbitrary, but each test is structured with this logic: `test_[method tested or method where the tested parameter is modified]_[parameter tested or more specific info about the test]`

Reply:
related to my next comment, I would prefer to keep the function names shorter and split the tests up into separate files. This doesnt only help in terms of keeping each file shorter and easier to manage, but also allows for easily testing all files related to one method (or group of methods). It's also easier to autocomplete in the terminal if the differences are in the filenames rather than the test names if one wants to run only one or a few tests.

If you agree, we can make a separate PR for reorganizing the test folder rather than doing it in this PR.

Reply:
Yes, let's do that